language/classify_text/classify_text_tutorial.py

```diff
@@ -0,0 +1,251 @@
+# Copyright 2017, Google, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START classify_text_tutorial]
+"""Using the classify_text method to cluster texts."""
+
+# [START classify_text_tutorial_import]
+import argparse
+import json
+import os
+
+from google.cloud import language_v1beta2
+from google.cloud.language_v1beta2 import enums
```

Comment:
you can just use `language_v1beta2.types` and `language_v1beta2.enums` if you want to save yourself the trouble of importing.

Reply:
In order to reduce clutter in other samples (e.g. prefer `types.Document` over `lanbuage_v1beta2.Document`) I have been importing all three separately.  I hope to keep it consistent in this code sample as well.